ansible/run-integration-tests.yml

```diff
@@ -21,6 +21,10 @@
         cmd: "docker login -u {{ quay_username }} --password-stdin quay.io"
         stdin: "{{ quay_password }}"
 
+    - name: Set tests as successful
+      set_fact:
+        success: true
```

Comment:
[nit] can we have a comment here about what this is for and how it can be modified? from this file alone it looks like the `Check results` step is never going to run